package/boot/uboot-envtools/files/lantiq

```diff
@@ -26,6 +26,9 @@ BTHOMEHUBV3A)
 P2812HNUF1)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x2000" "0x20000" "1"
 	;;
+VGV7519)
```

Comment:
why is it required to write the uboot env of the VGV7519 from userspace?

Side note: why is neither the u-boot nor the u-boot env partition of the VGV7519 write protected. Could you fix this as well.